doc/release-notes.md

```diff
@@ -4,36 +4,83 @@ release-notes at release time)
 Notable changes
 ===============
 
+Changes to Testnet NU5 Consensus Rules
+--------------------------------------
+
+- In order to better support hardware wallets, transparent signature hash construction as
+  defined in [ZIP 244](https://zips.z.cash/zip-0244) has been modified to include a hash
+  of the serialization of the amounts of all outputs being spent, along with a hash of all
+  spent outputs `scriptPubKeys` values, except in the case that the `ANYONECANPAY` flag is
+  set. This allows hardware wallet devices to verify the UTXO amounts without having to
+  stream all the previous transactions containing the outputs being spent to the device.
+  Also as part of these changes, the transparent signature hash digest now commits
+  directly, rather than implicitly, to the sighash type, and the sighash type is
+  restricted to a fixed set of valid values. The change to ZIP 244 can be seen 
+  [here](https://github.com/zcash/zips/commit/ac9dd97f77869b9191dc9592faa3ebf45475c341).
+- This release fixes a bug in `v4.6.0` that caused a consensus failure on the Zcash
+  testnet at height `1,779,200`. For details see
+  <https://github.com/zcash/zcash/commit/5990853de3e9f04f28b7a4d59ef9e549eb4b4cca>.
```

Comment:
Can we make this a link like:

For details see this [commit](https://github.com/zcash/zcash/commit/5990853de3e9f04f28b7a4d59ef9e549eb4b4cca)

Reply:
We certainly could, but I'm not sure that's particularly more helpful?